.github/ISSUE_TEMPLATE/documentation_request.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Documentation request
+about: Ask for documentation of a feature
+title: "Docs: ..."
+labels: type/docs
+assignees: ""
+---
+
+#### Is your documentation request related to a feature? Which one?
```

Comment:
Why is this `####` instead of `#`?
```suggestion
#### Is your documentation request related to a feature? If so, which one?
```

Reply:
We discussed `####` when changed the other issue templates: `#` is too big for what this represents, and is bigger than the title of the issue itself. We agreed that `####` was a reasonable side while still maintaining the header tag (as opposed to just bold text with `**` which comes by default from Github) for accesibility.